datalad_registry/templates/overview.html

```diff
@@ -36,33 +36,36 @@
           <tr>
             <th><a {{ sort_href('url-desc', 'url-asc') }}>URL</a></th>
             <th>Dataset</th>
-            <th>Commit</th>
-            <th><a {{ sort_href('keys-desc', 'keys-asc') }}>Annex keys</a></th>
-            <th>Annexed files in working tree</th>
-            <th>Size of Annexed files in working tree (bytes)</th>
-            <th><a {{ sort_href('update-desc', 'update-asc') }}>Last update</a></th>
-            <th>.git/objects Size (KiB)</th>
+              <th>Commit</th>
```

Comment:
why added indentation placing prior `<th>` on its own indentation level?
that introduces the diff which is not really about what this PR is about -- took time to figure out what actually changed to address original issue.

Next times try to not introduce diff unrelated to the fix, or place styling changes into a separate commit and alert about that (if PR consists of some limited number of commits and could be reviewed at commits levels)